app/android/app/build.gradle

```diff
@@ -44,8 +44,8 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.example.effektio"
-        minSdkVersion flutter.minSdkVersion
-        targetSdkVersion flutter.targetSdkVersion
+        minSdkVersion 21
+        targetSdkVersion 29
```

Comment:
what is this about?

Reply:
For generating the icons for android we need min SDK version as 21 or higher but the version here is lesser so did the changes, Flutter commands give error to these..